### PR TITLE
Test created by the C3 team.

### PR DESCRIPTION
**Pull request by marcosnc**

### CODE REVIEW

File: algorand/TmplSig.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def __init__(self, name):
 #            self.map = json.loads(f.read())
 
 
-        self.map = {"name":"lsig.teal","version":6,"source":"","bytecode":"BiABAYEASIAASDEQgQYSRDEZIhJEMRiBABJEMSCAABJEMQGBABJEMQkyAxJEMRUyAxJEIg==",
+        self.map = {"name":"lsig.teal","version":6,"source":"","bytecode":"BiABAYEASIAASDEQgQYSRDEZIhJEMRiBABJEMSCAABJEMQGBABJEMQkyAxJEMRUyAxJEIkM=",
```
Comment (jumpsiegel) [inline algorand/TmplSig.py:39]: We cannot change this... it will effect the hashes

Comment (noitatum) [reply algorand/TmplSig.py]: You are right, I reverted that change and modified the tests accordingly. Just beware that the disparity still exists.

**Comment by noitatum** [comment]
> ```python
> Traceback (most recent call last):
> (True, 'teal/core_approve.teal', 'teal/core_clear.teal', <algosdk.v2client.algod.AlgodClient object at 0x7f90d93f7070>, 1002000, <TmplSig.TmplSig object at 0x7f90db75a530>, True)
>   File "/home/runner/work/wormhole/wormhole/algorand/test/test.py", line 798, in <module>
>     core.simple_test()
>   File "/home/runner/work/wormhole/wormhole/algorand/test/test.py", line 520, in simple_test
>     self.vaa_verify = self.client.compile(get_vaa_verify())
> TypeError: get_vaa_verify() missing 1 required positional argument: 'file_name'
> Writing teal/core_approve.teal
> Writing teal/core_clear.teal
> ```
> 
> Need to fix test.py given the API change

I added a default parameter to get_vaa_verify

**Review by jumpsiegel — APPROVED** [review]

**Review by jumpsiegel — DISMISSED** [review]

File: algorand/audit_test/src/Wormhole.test.ts [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+	it('Can perform register chain', async () => {
+		// const CHAIN_ID_FOOBAR = 65000
+		// const EMITTER_FOOBAR = decodeBase16('ccddeeffccddeeffccddeeffccddeeffccddeeffccddeeffccddeeffccddeeff')
+		// const regVaa = wormhole.generateRegisterChainVAA(0, signers, 0, 0, 0, 0, CHAIN_ID_ALGORAND, CHAIN_ID_FOOBAR, EMITTER_FOOBAR)
+		// const signedVaa = wormhole.signVAA(regVaa)
+		// const txId = await wormhole.sendSignedVAA(signedVaa)
+		// await helper.waitForTransactionResponse(txId)
+	})
+
+	it('Can perform token transfer', () => {})
```
Comment (panoel) [inline algorand/audit_test/src/Wormhole.test.ts:161]: These are place holders for future tests?

**Review by panoel — APPROVED** [review]

**Review by jumpsiegel — APPROVED** [review]

**Review by panoel — APPROVED** [review]

**Review by panoel — APPROVED** [review]

**Pull request closed without merging**